models/concrete_index.go

```diff
@@ -72,8 +74,18 @@ func (i *ConcreteIndex) setName() {
 		return
 	}
 	// postgresql enforces a 63 character length limit on all identifiers
+	prefix := ""
+	switch i.Type {
+	case IndexTypeNavigation:
+		prefix = "nav"
```

Comment:
I added this prefix in [this interdiff](https://github.com/checkmarble/marble-backend/compare/ab6c9c037aea8de9a20a44d7fd2ee9634a875739..455bde4725a1897cdc045ce751ddd8786fa8b83b) of the index deletion backgroud task, so we do not have surprises when we continue and merge it.